jobs/pull-kubernetes-e2e-gke.sh

```diff
@@ -36,12 +36,12 @@ export KUBE_FASTBUILD=true
 
 # TODO(spxtr): once https://github.com/kubernetes/kubernetes/pull/35453 is in,
 # remove the first branch here.
-if [[ -e build/util.sh ]]; then
-  version=$(source build/util.sh && echo $(kube::release::semantic_version))
+if [[ -e build-tools/util.sh ]]; then
```

Comment:
we need to support both `build/` and `build-tools/`. this made both branches into `build-tools/`.